functests/4_latency/latency.go

```diff
@@ -375,13 +407,6 @@ func extractLatencyValues(testName string, exp string, node *corev1.Node) string
 // MAXIMUM_LATENCY: unified expected maximum latency for all tests
 func setMaximumLatencyValue(testName string) error {
```

Comment:
I think this should be converted into a getter as well to get rid of the `maximumLatency` variable

Reply:
fixed , thanks